includes/modules/order_total/ot_coupon.php

```diff
@@ -494,15 +494,16 @@ function calculate_deductions() {
       }
 //echo 'ot_coupon coupon_total: ' . $coupon->fields['coupon_calc_base'] . '<br>$orderTotalDetails[orderTotal]: ' . $orderTotalDetails['orderTotal'] . '<br>$orderTotalDetails[totalFull]: ' . $orderTotalDetails['totalFull'] . '<br>$coupon_total: ' . $coupon_total . '<br><br>$coupon->fields[coupon_minimum_order]: ' . $coupon->fields['coupon_minimum_order'] . '<br>$coupon_total_minimum: ' . $coupon_total_minimum . '<br>';
 // @@TODO - adjust all Totals to use $coupon_total but strong review for what total applies where for Percentage, Amount, etc.
-//      if ($coupon->RecordCount() > 0 && $orderTotalDetails['totalFull'] != 0) {
-      if ($coupon->RecordCount() > 0 && $orderTotalDetails['orderTotal'] != 0 ) {
+//      if ($coupon->RecordCount() > 0 && $orderTotalDetails['totalFull'] != 0)
+      if ($coupon->RecordCount() > 0 && $orderTotalDetails['orderTotal'] != 0 ) 
+      {
```

Comment:
Our current coding standards (psr2) state that opening braces for if statements should be on the same line. Given you didn't move the brace in `attributes.php` was it intentional here.

Reply:
My thinking was to move it down in places where there are commented out lines that may be preferred to the current uncommented out line.  But I could move it back to the current live line if preferred; people would just have to remember to add it  to the commented out line if they uncomment it.

Reply:
Yes, would move it back. I guess as well I'm not a great fan of putting alternate code into comments any way.